### ISSUE TITLE
About classification

### ISSUE THREAD

**Issue by laiyingxin2** [caoyunkang/GroundedSAM-zero-shot-anomaly-detection#12]
Thank you so much for your wonderful paper! I have really enjoyed it! 
I have a question regarding the SSA,
1.  What is the model's classification result for abnormal/non-abnormal in the entire picture?
2.  What is the cross-domain effect of classification?
3.  I noticed that adding a classification loss can lead to overfitting and non-convergence during fine-tuning. Could you please share any approaches you have used to address this problem?
Thank you very much for your time and expertise. I greatly appreciate any insights or suggestions

**Comment by caoyunkang** [comment]
Hi! Thanks for your interest. Much to our regret, SAA's classification performance is subpar as it always selects the most significant regions to be anomalies, i.e., it tends to be false-alarmed. You could directly run our code to evaluate the classification performance. Best :)
